README.md

```diff
@@ -33,12 +33,20 @@ First, dump your MySQL database in PostgreSQL-compatible format
     mysqldump --compatible=postgresql --default-character-set=utf8 \
     -r databasename.mysql -u root gitlabhq_production -p
 
-Then, convert it using the dbconverter.py script
+Then, convert it using the dbconverter.py script.
 
-`python db_converter.py databasename.mysql databasename.psql`
+    python db_converter.py databasename.mysql databasename.psql
 
 It'll print progress to the terminal
 
+Now we have a DB dump that can be imported but the dump will be slow due
+to existing indexes. We use 'ed' to edit the DB dump file and move the
+'DROP INDEX' statements to the start of the import. Ed is not the fastest
+tool for this job if your DB dump is multiple gigabytes. (Patches to
+the converter are welcome!)
+
+    ed -s databasename.psql < move_drop_indexes.ed
```

Comment:
It would be nicer to handle all of this from the db_converter.py script but that would mean dropping support for writing to stdout (not a big deal) so that we can rewind the file and insert the 'DROP INDEX' lines at the start of it. For the moment that is too much python hacking for me.